apps/docs/src/content/docs/guides/migration.mdx

```diff
@@ -0,0 +1,308 @@
+---
+title: Migration
+---
+
+The preceding documentation provides developers with insights into transitioning from a React Native CLI project to one augmented by Flagship Code, facilitating ephemeral native code generation. However, this resource may prove less pertinent for developers transitioning from Flagship Code v12 to v13. While the overarching concepts surrounding native code generation, plugin integration, and typed configuration remain consistent, notable refinements and adjustments characterize the details and implementation of the updated iteration.
+
+Flagship Code has undergone a migration to a more adaptable project structure, featuring guarded configurations, generators, TypeScript-exported plugins, and split-configurations.
+
+## Dependencies
+
+The key highlight of the latest version of Flagship Code with respect to dependencies is its support for the most recent stable release of React Native, version 0.72.+. In adherence to industry best practices, we designate the stable version as the latest version minus one, recognizing that third-party libraries require time to align with the latest release.
+
+In this update, all Flagship Code dependencies have undergone a major version upgrade, accompanied by notable deprecations. Particularly, the package `@brandingbrand/code-core` has been deprecated in favor of `@brandingbrand/code-cli-kit`. This change stems from the recognition that the former package name lacked clarity regarding its intended use-case. We believe that `code-cli-kit` better communicates that this package serves as a developer toolkit to support the CLI package.
+
+Furthermore, several plugins have been deprecated due to their inability to undergo open-source testing. Some of these plugins necessitated third-party accounts, posing a risk of exposing Branding Brand's accounts to public developers. These deprecated plugins, including `@brandingbrand/code-plugin-fbsdk-next`, `@brandingbrand/code-plugin-firebase-analytics`, `@brandingbrand/code-plugin-firebase-app`, `@brandingbrand/code-plugin-google-signin`, and `@brandingbrand/code-plugin-leanplum`, will need to be re-implemented in the new plugin format, following the guidelines outlined in the [plugin guide](/guides/plugins). While we understand that this may be frustrating for developers, we firmly believe that this approach is necessary to ensure the quality and supportability of our plugins.
```

Comment:
@crherman7 @jasonmosley I think we may need discuss these removal of plugins and current usage. My personal thought is we may need to keep them for this version and then build out a true deprecation path rather then just removing from this version.

Reply:
@NickBurkhartBB @crherman7 I agree. We should mark them as deprecated but recreate them for this version. We could even have part of the migration plan move these into the projects as local plugins. Since the old plugins are not compatible with this version, we can't leave existing clients hanging.